libs/langgraph/langgraph/prebuilt/tool_node.py

```diff
@@ -168,8 +170,23 @@ def tool_node(state: dict):
         return {"messages": result}
     ```
 
+    Tool calls can also be passed directly to a ToolNode. This can be useful when using
+    the Send API, e.g., in a conditional edge:
+
+    ```python
+    def example_conditional_edge(state: dict) -> List[Send]:
+        tool_calls = state["messages"][-1].tool_calls
+        # If tools rely on state or store variables (whose values are not generated
+        # directly by a model), you can inject them into the tool calls.
+        tool_calls = [
+            tool_node.inject_tool_args(call, state, store)
+            for call in last_message.tool_calls
+        ]
+        return [Send("tools", [tool_call]) for tool_call in tool_calls]
+    ```
+
     Important:
-        - The state MUST contain a list of messages.
+        - The input state MUST contain either a list of messages or a list of tool calls.
```

Comment:
nice update -- i think we can go a bit further and explain that inputs can be one of 3 things:
- dict w/ messages_key -> list of messages
- list of messages
- list of tool calls